spec/features/sign_in_out_spec.rb

```diff
@@ -26,7 +26,8 @@
           click_button 'Sign in'
         end
 
-        expect(page).to have_content(@user.username || @user.email)
+        expect(current_url) == '/users/:id'
+        expect(page).to have_content(@user.username)
```

Comment:
I wrote `expect(page).to have content(@user.name || @user.email)` for the "user signs in and out" story.  Prior to starting the "user show" story, I modified that test and implemented `expect(current_url) == '/user/:id'`.